org.mixxx.Mixxx.yaml

```
@@ -44,175 +56,291 @@ cleanup:
       - /bin
 
+  # USB and Bluetooth HID library
   - name: hidapi
     buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
     sources:
       - type: archive
-        url: https://github.com/libusb/hidapi/archive/refs/tags/hidapi-0.12.0.tar.gz
-        sha256: 28ec1451f0527ad40c1a4c92547966ffef96813528c8b184a665f03ecbb508bc
+        url: https://github.com/libusb/hidapi/archive/refs/tags/hidapi-0.14.0.tar.gz
+        sha256: a5714234abe6e1f53647dd8cba7d69f65f71c558b7896ed218864ffcf405bcbd
 
+  # Audio metadata library
   - name: taglib
     buildsystem: cmake-ninja
     config-opts:
+      # Enable shared libraries and disable tests
       - -DBUILD_SHARED_LIBS=ON
-      - -DWITH_MP4=ON
-      - -DWITH_ASF=ON
+      - -DBUILD_TESTING=OFF
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
     sources:
       - type: archive
-        url: https://taglib.org/releases/taglib-1.12.tar.gz
-        sha256: 7fccd07669a523b07a15bd24c8da1bbb92206cb19e9366c3692af3d79253b703
+        url: https://taglib.org/releases/taglib-1.13.1.tar.gz
+        sha256: c8da2b10f1bfec2cd7dbfcd33f4a2338db0765d851a50583d410bacf055cfd0b
     cleanup:
       - /bin
 
+  # Audio fingerprinting library
   - name: chromaprint
     buildsystem: cmake-ninja
     config-opts:
+      # Disable command line tools and tests
+      - -DBUILD_TOOLS=OFF
+      - -DBUILD_TESTS=OFF
       - -DCMAKE_BUILD_TYPE=RelWithDebInfo
-      - -DBUILD_SHARED_LIBS=ON
     sources:
       - type: archive
         url: https://github.com/acoustid/chromaprint/releases/download/v1.5.1/chromaprint-1.5.1.tar.gz
         sha256: a1aad8fa3b8b18b78d3755b3767faff9abb67242e01b478ec9a64e190f335e1c
 
+  # Audio time stretching and pitch shifting library
   - name: rubberband
     buildsystem: meson
+    config-opts:
+      # Disable tests, command line tools and non-functional audio plugins
+      - -Dtests=disabled
+      - -Dcmdline=disabled
+      - -Dladspa=disabled
+      - -Dlv2=disabled
     sources:
       - type: archive
-        url: https://github.com/breakfastquay/rubberband/archive/refs/tags/v2.0.2.tar.gz
-        sha256: 1372a0d52915ef07aa3cffc00a8b17996c736f328efa0f6504d231fa6428ed4b
+        url: https://github.com/breakfastquay/rubberband/archive/refs/tags/v3.3.0.tar.gz
+        sha256: 2bb837fe00932442ca90e185af8a468f7591df0c002b4a9e27a1bced1563ac84
 
+  # Audio processing Library
   - name: soundtouch
     buildsystem: cmake-ninja
+    config-opts:
+      # Disable soundstretch utility and disable non-working -mfpu=neon flag
+      - -DSOUNDSTRETCH=OFF
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DNEON=OFF
     sources:
       - type: archive
-        url: https://codeberg.org/soundtouch/soundtouch/archive/2.3.1.tar.gz
-        sha256: 42633774f372d8cb0a33333a0ea3b30f357c548626526ac9f6ce018c94042692
+        url: https://codeberg.org/soundtouch/soundtouch/archive/2.3.2.tar.gz
+        sha256: 714c8626001bb3ef291a9af9eadc0f7b382a0b23506efebc516b63b170b4da56
```

```suggestion
        sha256: ed714f84a3e748de87b24f385ec69d3bdc51ca47b7f4710d2048b84b2761e7ff
```

Codeberg changed the hash :/ Probably didn't break CI here yet due to caching.